src/lib/cookies/cookiemanager.cpp

```diff
@@ -108,17 +109,19 @@ void CookieManager::removeAll()
     ui->cookieTree->clear();
 }
 
-void CookieManager::removeCookie()
+// Return the domain of the removed cookie
+QString CookieManager::removeCookie()
```

Comment:
Instead of changing the `removeCookie()` method, there should be new method to return domain of current cookie.
Or, as it is only used in one place, it may not even be made to function.